src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -187,6 +180,24 @@ def register_options(cls, register):
 
   def register_extra_products_from_contexts(self, targets, compile_contexts):
     super(RscCompile, self).register_extra_products_from_contexts(targets, compile_contexts)
+
+    def cp_entry_for(maybe_file):
+      # Capture a snapshot for the file if it exists
+      if os.path.exists(maybe_file):
+        def capture_or_read_digest(scheduler, filename):
+          digest = load_digest(os.path.dirname(filename))
```

Comment:
I don't have a feel for it. This was a step towards just consuming the digests directly from each tool invoke's output directory. There may be some classpath entries that are missing digests, and I didn't spend the time to track down which ones and fix them as part of the initial cut of this change.

Reply:
You were right that the digests weren't being loaded. They are now. The reason I'm dropping them in files is because I needed to access the digests from different template methods. I could instead hold them in a instance variable, but then they wouldn't be reused across runs. Maybe that's something I shouldn't worry about right now?

I agree that we should probably come up with a common approach to digest files if we're going to use them.